Assets/Scripts/Map/Partition.cs

```diff
@@ -38,30 +38,57 @@ public sealed class Partition
         public int PartitionId { get; }
         public IReadOnlyList<Vertex> Vertices { get; }
         public Bitmap CommunicationZone { get; }
-        public IReadOnlyDictionary<Vector2Int, Bitmap> CommunicationZones { get; }
+        public IReadOnlyDictionary<Vector2Int, Bitmap> WaypointsCommunicationZones { get; }
         public IReadOnlyDictionary<int, Bitmap> IntersectionZones => _intersectionZones;
         public IReadOnlyDictionary<int, float> CommunicationRatio => _communicationRatio;
+        public List<Partition> NeighborPartitions { get; } = new();
 
         private readonly Dictionary<int, Bitmap> _intersectionZones = new();
         private readonly Dictionary<int, float> _communicationRatio = new();
+        private readonly int _bitmapWidth;
+        private readonly int _bitmapHeight;
```

Comment:
Why are we storing this?

Reply:
We need it when creating an instance of the intersecting zone where we have waypoints in the zone